amundsen-kube-helm/templates/helm/values.yaml

```diff
@@ -83,6 +124,12 @@ search:
   # search.podAnnotations -- Search pod specific annotations.
   podAnnotations: {}
 
+  # Everything set under "envVars" during helm install/upgrade will be parsed as environment variable for this service.e
+  envVars:
+    # No need to explicitly add the variables in values file or in configmap. You can pass these variables runtime
+    FLASK_OIDC_CLIENT_ID: ""
```

Comment:
it doesn't make sense to set those at all if users are not using OIDC

Reply:
The reason I did put it here is to document it properly as to where/how to set these variables. 
I'll be moving these OIDC related stuff under oidc property, and that will remove the confusion.

Reply:
fixed.